python/src/KarhunenLoeveAlgorithmImplementation_doc.i.in

```diff
@@ -73,9 +86,10 @@ where :math:`\vect{\phi}_{pk} \in \Rset^d`. We note:
         \right) \in \Rset^{Pd}
      \end{align*}
 
-and :math:`\mat{\vect{\Phi}} = (\vect{\Phi}_1\, |\, \dots \, | \vect{\Phi}_K)` the matrix of the :math:`K` first modes of the Karhunen-Loeve decomposition.
+and :math:`\mat{\vect{\Phi}} = (\vect{\Phi}_1\, |\, \dots \, | \vect{\Phi}_K)` the
+matrix of the :math:`K` first modes of the Karhunen-Loeve decomposition.
 
-The approximated Fredholm problem writes for all :math:`k \geq 1`:
+The approximated Fredholm problem is for all :math:`k \geq 1`:
```

Comment:
Suggested change below:

"The Fredholm problem writes:" --> "The Fredholm problem can be expressed as:"

Reply:
Ok. Fixed at acd7c674c1e3510d9f7a9af01ee0a97a94b193ec